packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2874,7 +2874,15 @@ class InputDecoration {
 
   /// The maximum number of lines the [helperText] can occupy.
   ///
-  /// Defaults to null, which means that the [helperText] is not limited.
+  /// Defaults to null, which means that soft line breaks in [helperText] are
+  /// truncated with an ellipse while hard line breaks are respected.
+  /// For example, an [helperText] that overflows the width of the field will be
+  /// truncated with an ellipse. However, an [helperText] with explicit linebreak
+  /// characters (\n) will display on multiple lines.
+  ///
+  /// To get a long [helperText] wrapping, either set [helperMaxLines] or use
```

Comment:
Nit, maybe this sounds better: "To get a long [helperText] wrapping" => "To cause a long [helperText] to wrap"

Same for errorText below.